factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/registry/transformation/chains/TransformationChains.java

```diff
@@ -128,10 +128,9 @@ private TransformationChain build(TransformationKey key, int from, int to)
         }
         List<Transformation> steps = map(path, Edge::transformation);
 
-        int cost = ((Double) ((HeuristicNode) r.getGoalNode()).getCost()).intValue();
+        int cost = ((Double) ((HeuristicNode<?, ?, ?, ?>) r.getGoalNode()).getCost()).intValue();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused local variables such as 'cost'.](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=5227401)

Reply:
🤔no cost calculations anymore?

Reply:
removed cost attribute from transformationchain. cost consideratoin is taken care of by the algorithm itself

Reply:
this line can therefore also be removed

Reply:
Got it :)